scripts/inline_comments_data_collection.py

```diff
@@ -136,21 +104,33 @@ def process_comments(limit, diff_length_limit):
             if not most_recent_update:
                 continue
 
-            fix_patch_id, revision_phid = get_diff_info_from_phid(
-                most_recent_update["fields"]["new"]
-            )
-
-            # If the most recent patch is the original patch itself, skip it
-            if fix_patch_id == patch_id:
+            try:
+                fix_patch_id = diff_phid_to_id[most_recent_update["fields"]["new"]]
+            except KeyError:
+                diffs = api.search_diffs(diff_phid=most_recent_update["fields"]["new"])
+                if not diffs:
+                    raise NoDiffFoundForPHIDException(
+                        most_recent_update["fields"]["new"]
+                    )
+                fix_patch_id = diffs[0]["id"]
+
+            # If the most recent patch doesn't exist or is the original patch itself, skip it
+            if not fix_patch_id or fix_patch_id == patch_id:
```

Comment:
Now `not fix_patch_id` is no longer needed, right?

Reply:
Correct, removed here: 8ac58d6720b832afff9cba22242867231a80a3eb